.travis.yml

```diff
@@ -0,0 +1,24 @@
+sudo: required
+
+language:
+   - python
+
+services:
+   - docker
+
+env:
+   global:
+      - CLEAN_TRAVIS_TAG=${TRAVIS_TAG/[[:space:]]/}
+      - COMMIT=${CLEAN_TRAVIS_TAG:-${TRAVIS_COMMIT:0:7}}
+
+script:
+   - .travis/build
+
+after_success:
```

Comment:
Why use `after_success` and not `deploy` here?

Reply:
I based it off of an example that used `after_success`. Is there a benefit to using `deploy` instead?

Reply:
Seems like `deploy` is specifically for deploying to something like heroku. https://docs.travis-ci.com/user/customizing-the-build#Deploying-your-Code

I'm not sure it works against an unsupported provider, which quay.io is. List is here: https://docs.travis-ci.com/user/deployment/#Supported-Providers